watertap/unit_models/ion_exchange_0D.py

```diff
@@ -599,539 +650,668 @@ def build(self):
 
         self.Pe_p = Var(
```

Comment:
```suggestion
        self.N_Pe_particle = Var(
```

Reply:
modified all dimensionless numbers to match this format